site/public/css/email-status.css

```diff
@@ -0,0 +1,40 @@
+.btn{
```

Comment:
This is going to collide with the `.btn` class from bootstrap. Please try and avoid re-using class names from bootstrap as it's going to create weird collisions and developer misunderstandings.

Reply:
Renamed the .btn class in css to .status-btn, and added the .status-btn class to the button on the page.